### PR TITLE
docs: how to use metric hub with mozanalysis

### PR DESCRIPTION
**Pull request by rebecca-burwei**

### CODE REVIEW

File: src/concepts/metric_hub.md [hunk 1]
```diff
@@ -147,3 +147,10 @@ config_collection = ConfigCollection.from_github_repo("https://github.com/mozill
 metric = config_collection.get_metric_definition(slug="active_hours", app_name="firefox_desktop")
 print(metric.from_expression)
 ```
+
+To use the metrics with Mozanalysis, you'll need `Metric`s not `MetricDefinition`s. For example:
```
Comment (scholtzan) [inline src/concepts/metric_hub.md:151]:
```suggestion
To use the metrics with Mozanalysis, you'll need `Metric`s not `MetricDefinition`s. For example:

```

this should fix the CI failure

**Review by scholtzan — APPROVED** [review]

File: src/concepts/metric_hub.md (hunk 1 above)
Comment (rebecca-burwei) [reply src/concepts/metric_hub.md:151]: thank you!

File: src/concepts/metric_hub.md [hunk 2]
```diff
@@ -147,3 +147,11 @@ config_collection = ConfigCollection.from_github_repo("https://github.com/mozill
...
+
+```python
+from mozanalysis.config import ConfigLoader
+metric = ConfigLoader.get_metric(slug="active_hours", app_name="firefox_desktop")
+```
+
```
Comment (badboy) [inline src/concepts/metric_hub.md:157]:
```suggestion
```

and one line less at the end (gosh, prettier is picky...)

Comment (rebecca-burwei) [reply src/concepts/metric_hub.md:157]: thank you!

**Pull request merged**